cfgov/regulations3k/models/pages.py

```diff
@@ -98,8 +103,13 @@ def regulation_results_page(self, request):
             pre_tags=['<strong>'],
             post_tags=['</strong>']).models(SectionParagraph)
         for hit in sqs:
+            try:
+                snippet = Markup(" ".join(hit.highlighted))
+            except TypeError as e:
+                logger.info("Query string {} produced a TypeError: {}".format(
```

Comment:
How would you feel about using `logger.warning` here instead?

Reply:
Good idea.